styles.css

```diff
@@ -0,0 +1,89 @@
+body {
+  font-family: sans-serif;
+  /* background-color: brown; */
+  margin: 0;
+  padding: 0;
+}
+
+.main-container {
+  display: flex;
+  flex-direction: column;
+  gap: 30px;
+  /* background-color: blue; */
+  padding: 25px;
+  margin: auto;
+  width: 80%;
+  text-align: center;
+}
+
+@media (min-width: 1031px) {
+  .main-container {
+      text-align: left;
+  }
+}
+
+h1 {
+  font-size: 38px;
+  margin: 0;
+  padding: 0;
+  font-family: 'Ubuntu', sans-serif;
+  font-weight: 500;
+}
+
+.products {
+  /* background-color: aqua; */
+  display: grid;
+  grid-template-columns: repeat(auto-fit, 255px);
+  grid-gap: 30px;
+  justify-content: center;
+}
+
+.product {
+  /* background-color: chartreuse; */
+  display: flex;
+  flex-direction: column;
+  /* align-items: center; */
+  gap: 15px;
+  padding: 28px;
+}
+
+.product:hover {
+  transform: scale(1.05);
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.07), 0 2px 4px rgba(0, 0, 0, 0.03);
+}
```

Comment:
You can use cursor pointer here, so it can be more interactive with the user.

Other thing, in this case is not necessary because this has no logic, but in general, this should be a button to have more accessibility, because at the end of the day, this is a button that onClick should open the details of a product.